src/components/inputModal.tsx

```diff
@@ -0,0 +1,33 @@
+import { Modal, Button } from 'react-bootstrap'
+import React from 'react'
+import { useState } from 'react'
```

Comment:
it's better to extract things from common module in one statement
```suggestion
import React, { useState } from 'react'
```